deploy/helm/kuberhealthy/templates/servicemonitor.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     app: {{ template "kuberhealthy.name" . }}
     prometheus: {{ .Values.prometheus.name }}
     release: {{ .Values.prometheus.release }}
+    {{- if .Values.prometheus.serviceMonitor.additional_labels}}
+    additional_labels: {{ .Values.prometheus.serviceMonitor.additional_labels}}
+    {{- end  -}}
```

Comment:
can we change it to use camelCase? I believe thats part of the helm best practices: https://helm.sh/docs/chart_best_practices/values/#naming-conventions

Reply:
Done!